tests/system/providers/amazon/aws/rds/example_rds_db_instance.py

```diff
@@ -0,0 +1,75 @@
+#
```

Comment:
One of the convention we (AWS) are following is we name the file with the same name we use as dag ID. Here `example_rds_instance` makes sense so we should rename this file `example_rds_instance`

Reply:
Ah my bad, overlooked it, will fix.

Reply:
The file is renamed.